rich/logging.py

```diff
@@ -56,7 +58,9 @@ def __init__(
         super().__init__(level=level)
         self.console = console or get_console()
         self.highlighter = highlighter or self.HIGHLIGHTER_CLASS()
-        self._log_render = LogRender(show_level=True, show_path=show_path)
+        self._log_render = LogRender(
```

Comment:
The `LogRender` class is also use in Console when you call `console.log` -- which doesn't have a `level` argument.

Reply:
Ok, I've removed the log level from the creation of a console.